antarest/login/model.py

```diff
@@ -67,7 +67,7 @@ class UserCreateDTO(AntaresBaseModel):
 
 
 class GroupDTO(AntaresBaseModel):
-    id: Optional[str] = None
```

Comment:
Actually this changes the contract of the group creation endpoint:
it will now fail if we don't provide an ID.

We need either to change this endpoint to use another class where id is optional, and ensure we create an ID (I think this is automatic when we save to database, it creates a UUID), or ensure that we call the endpoint with an ID everywhere.

(note: this is why the python test fail, but actually the calls from the group creation dialog will also fail now)